.buildkite/lint.rayci.yml

```diff
@@ -86,12 +86,25 @@ steps:
     commands:
       - ./ci/lint/check-documentation-style.sh
 
-  - label: ":lint-roller: lint: linkcheck"
+  # premerge linkcheck run; validate links within the ray repository
+  - label: ":lint-roller: lint: ray linkcheck"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b"
```

Comment:
Might be good to leave a comment saying what pipeline this UUID is for - others may not know why this only runs on one particular pipeline.

Reply:
I added comments in the change but maybe it's too hidden or you think it's better to move it to the if statement

`# premerge linkcheck run; validate links within the ray repository`